# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../glibc/x86-i686.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

__i686 is used in assembler, but when optimizing for a 686-class CPU
gcc defines __i686 as cpp macro ...

  - Rene Rebe <rene@exactcode.de>

--- glibc-20050919/csu/defs.awk.vanilla	2005-11-03 13:16:56.000000000 +0100
+++ glibc-20050919/csu/defs.awk	2005-11-03 13:17:10.000000000 +0100
@@ -24,4 +24,5 @@
 
     print "#include <libc-symbols.h>";
     print "weak_extern (__gmon_start__)";
+    print "#undef __i686";
 }
